cocotb/_build_libs.py

```diff
@@ -148,6 +142,8 @@ def _extra_link_args(lib_name=None, rpath=None):
     if sys.platform == "darwin" and lib_name is not None:
         args += ["-Wl,-install_name,@rpath/%s.so" % lib_name]
```

Comment:
So based on https://github.com/cocotb/cocotb/pull/1714#issuecomment-619625219 we can remove those also?

Reply:
We need to keep it. The default `install_name` is the path where the library got written by the compiler which will be some temporary build directory that pip uses.